tomcat-gcp-lib/src/main/java/com/google/cloud/runtimes/tomcat/session/DatastoreValve.java

```diff
@@ -64,7 +67,25 @@ public void invoke(Request request, Response response) throws IOException, Servl
       } else {
         log.error("In order to persist the session the manager must implement StoreManager");
       }
+    } else {
+      log.debug("Session not persisted");
```

Comment:
It would be helpful to include the reason the session is not persisted. "Session not persisted because URI /foo/bar is excluded".